docs/direct_agent_tutorial.md

```diff
@@ -0,0 +1,388 @@
+# Direct Agent Tutorial
```

Comment:
You also explain how to call an agent using http, so you could change your title and name of the file to be more generic